src/main/java/io/vertx/cassandra/CassandraClientOptions.java

```diff
@@ -55,6 +53,11 @@ public CassandraClientOptions setPort(int port) {
     return this;
   }
 
+  public CassandraClientOptions addContactPoint(String address) {
+    contactPoints().add(address);
+    return this;
+  }
+
   public List<String> contactPoints() {
     return contactPoints;
```

Comment:
Make sure to initialize `contactPoints` to an empty `ArrayList` if this method is invoked and the field is null

Reply:
Why we can't initialize `contactPoints` with an empty `ArrayList` when `CassandraClientOptions` is created?

Reply:
ah, i got it

Reply:
Done